_posts/0000-01-02-kanelushi.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![Captura de Pantalla 2020-11-18 a la(s) 21 15 47](https://user-images.githubusercontent.com/74990520/100154376-3b46f980-2e84-11eb-8c6d-424858f787db.png)

Reply:
django: conectar templates to views. model<=>views. views to urls

Reply:
```suggestion
Your text
```